internal/controller/operatorconfigmap_controller.go

```diff
@@ -411,14 +412,25 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI(storageClients *v1al
 	}
 
 	cniNetworkAnnotationValue := ""
+	topologyKey := []string{}
 	for i := range storageClients.Items {
 		if annotationValue := storageClients.Items[i].GetAnnotations()[cniNetworksAnnotationKey]; annotationValue != "" {
 			if cniNetworkAnnotationValue != "" {
 				return fmt.Errorf("only one client with CNI network annotation value is supported")
 			}
 			cniNetworkAnnotationValue = annotationValue
 		}
+
+		// merge topology keys from all storage clients,
+		// as multiple storage clients hubs can have different topology keys
+		topologyKeyValue := storageClients.Items[i].GetAnnotations()[utils.AnnotationNonResilientPoolsTopologyKey]
```

Comment:
Can you please refactor the current item into its own var at the start of the loop and use it instead?
``` storageClient := &storageClients[i]```

Reply:
Done